modules/conflict_resolver/php/NDB_Menu_Filter_Form_conflict_resolver.class.inc

```diff
@@ -84,29 +86,30 @@ class NDB_Menu_Filter_Form_Conflict_Resolver extends NDB_Menu_Filter_Form
                         "SELECT UserID FROM $row[TableName] WHERE CommentID=:CID",
                         array('CID' => $row['CommentId2'])
                     );
-                    $resolutionLog =array(
-                                     'UserID'         => $user->getUsername(),
-                                     'User1'          => $user1,
-                                     'User2'          => $user2,
-                                     'TableName'      => $row['TableName'],
-                                     'ExtraKeyColumn' => $row['ExtraKeyColumn'],
-                                     'ExtraKey1'      => $row['ExtraKey1'],
-                                     'ExtraKey2'      => $row['ExtraKey2'],
-                                     'FieldName'      => $row['FieldName'],
-                                     'CommentId1'     => $row['CommentId1'],
-                                     'CommentId2'     => $row['CommentId2'],
-                                     'OldValue1'      => $row['Value1'],
-                                     'OldValue2'      => $row['Value2'],
-                                     'NewValue'       => $val,
-                                     'ConflictID'     => $row['ConflictID'],
-                                    );
+                    $resolutionLog = array(
+                                      'UserID'         => $user->getUsername(),
+                                      'User1'          => $user1,
+                                      'User2'          => $user2,
+                                      'TableName'      => $row['TableName'],
+                                      'ExtraKeyColumn' => $row['ExtraKeyColumn'],
+                                      'ExtraKey1'      => $row['ExtraKey1'],
+                                      'ExtraKey2'      => $row['ExtraKey2'],
+                                      'FieldName'      => $row['FieldName'],
+                                      'CommentId1'     => $row['CommentId1'],
+                                      'CommentId2'     => $row['CommentId2'],
+                                      'OldValue1'      => $row['Value1'],
+                                      'OldValue2'      => $row['Value2'],
+                                      'NewValue'       => $val,
+                                      'ConflictID'     => $row['ConflictID'],
+                                      'ProjectID'      => $row['ProjectID'],
```

Comment:
I don't think there's any reason to add this to the conflicts_resolved table. The CommentID(s) are already logged, and the project can be retrieved by joining if necessary. This seems like unnecessary denormalization.